src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/HistoryMapper.java

```diff
@@ -186,4 +180,38 @@ protected String mapPath(FlowPath path) {
             return null;
         }
     }
+
+    public long mapTimestamp(Instant value) {
+        return value.getEpochSecond();
+    }
+
+    /**
+     * Map {@link Cookie} into {@link long}.
+     */
+    public long mapCookie(Cookie value) {
+        if (value == null) {
+            return 0;
```

Comment:
Before, null cookie was mapped to null Long. What is the reason to implicitly introduce 0 as equivalent to null cookie? If null cookie is not acceptable then throw an exception.

Reply:
Reasoin is the possible NPE - target objects use "long" not "Long" in corresponding fields.